en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md

```diff
@@ -149,27 +177,31 @@ Now your REST API proxy is ready to be consumed. An application developer can di
 
 1. On the **Lifecycle Management** page, click **Go to DevPortal**.
 
-    ![Go to Developer Portal](../../assets/img/tutorials/go-to-devportal.png){.cInlineImage-threeQuarter}
+    ![Go to Developer Portal](../../assets/img/tutorials/proxy/go-to-devportal.png){.cInlineImage-full}
 
-2. Once you sign in to the API Developer Portal, click **Credentials** in the navigator.
+2. Once you open the API Developer Portal, click **Credentials** in the left navigation menu.
 
-3. Click **Generate Credentials**. As a result, Choreo populates the **Consumer Key** and **Consumer Secret** fields with the newly generated tokens.
+3. Click **Generate Credentials**. Choreo populates the **Consumer Key** and **Consumer Secret** fields with newly generated tokens.
 
 4. Click **://Curl**. Then copy the cURL command that is displayed.
 
     !!!info
-        Click the button to copy the cURL command. **Do not manually copy it**. The displayed cURL command is only a template, and it does not include the generated credentials.
+        Click the icon for copying to copy the cURL command. **Do not manually copy it**. The displayed cURL command is only a template, and it does not include the generated credentials.
+
+5. Close the **Get cURL to Generate Access Token** card. Then click **Update**.
```

Comment:
Let's mention that this way is solely for testing purposes. And for prod, we have to use the token endpoint with a suitable grant type.

Reply:
Done

Reply:
Let's remove this.